CHANGELOG.md

```diff
@@ -1,19 +1,25 @@
-## 2.1.0 (February 5, 2019)
+## 2.1.0 (March 25, 2019)
```

Comment:
👍 for the optimism!

Reply:
I suggest - as per  https://www.terraform.io/docs/extend/best-practices/versioning.html#changelog-specification
> NOTE: For HashiCorp released providers, the release process will replace the "Unreleased" header with the current date. This line must be present with the target release version to successfully release that version.

```suggestion
## 2.1.0 (Unreleased)
```

Reply:
Ok. For now Unreleased :)